src/pages/iou/steps/IOUParticipantsPage/IOUParticipantsRequest.js

```diff
@@ -71,6 +67,22 @@ class IOUParticipantsRequest extends Component {
         };
     }
 
+    /**
+     * @param {string} searchTerm
+     * @returns {Object}
+     */
+    getRequestOptions(searchTerm = '') {
+        return OptionsListUtils.getNewChatOptions(
+            this.props.reports,
+            this.props.personalDetails,
+            this.props.betas,
+            searchTerm,
+            [],
+            CONST.EXPENSIFY_EMAILS,
+            this.props.iouType !== CONST.IOU.MONEY_REQUEST_TYPE.SEND,
```

Comment:
Updated it in the doc, this should be like this:
```suggestion
            this.props.iouType === CONST.IOU.MONEY_REQUEST_TYPE.REQUEST,
```

@marcaaron Also could we add some explanation to the the `getRequestOptions` request doc why is it true only for request? thank you!

Reply:
Sure. This is the same logic though, right?

>could we add some explanation to the the getRequestOptions request doc why is it true only for request?

Maybe better to add that context here - can add something

Reply:
Thank you!